serverlessworkflow/modules/ROOT/pages/core/custom-functions-support.adoc

```diff
@@ -537,7 +537,7 @@ You must declare a `functionRef` like the following: (Do not forget to set `asCl
 
 [NOTE]
 ====
-{product_name} generates a CloudEvent ID based on the `source` and the workflow instance ID. In case an ID is set, {product_name} will ignore it and use a generated one.
+{product_name} generates a CloudEvent ID based on the `source` and the workflow instance ID. In case you decide to set an ID, {product_name} will use it and you must ensure it's unique.
```

Comment:
Are we explaining how to set such an id?

Reply:
Now we are :)